audio/speech/getting-started/get_started_with_chirp_2_sdk.ipynb

```diff
@@ -669,7 +671,13 @@
       "source": [
         "## Transcribe using Chirp 2\n",
         "\n",
-        "You can use Chirp 2 to transcribe audio in streaming, real-time and batch modes. In the following sections, you explore how to use the Speech-to-Text v2 API to transcribe audio in these three different scenarious."
+        "You can use Chirp 2 to transcribe audio in Streaming, Synchronous and Batch modes:\n",
+        "\n",
+        "* Streaming mode is good for streaming and real-time audio.  \n",
+        "* Synchronous mode is good for short audio < 1 min.\n",
```

Comment:
```suggestion
        "You can use Chirp 2 to transcribe audio in Streaming, Online and Batch modes:\n",
        "\n",
        "* Streaming mode is good for streaming and real-time audio.  \n",
        "* Online mode is good for short audio < 1 min.\n",
```

Not sure what the docs say, but I've been using Online/Batch terminology to match Vertex AI Online/Batch Prediction

Reply:
The documentation would mentioned about Synchronous but I agree that Online or even Real-time is better.